web/admin/src/Questions.js

```diff
@@ -38,7 +38,7 @@ export default class Questions extends PureComponent {
                       <div className="question-buttons">
                         { updater.state.hasPendingChanges && <button onClick={() => this.saveQuestion(item, updater)}>Save Question</button> }
                         { updater.state.hasPendingChanges && <button className="secondary" onClick={() => updater.cancel()}>Cancel Changes</button> }
-                        <button className="tertiary" onClick={this.deleteQuestion(item)}>Delete Question</button>
+                        <button className="tertiary" disabled={this.props.currentIndex === index} onClick={this.deleteQuestion(item)}>Delete Question</button>
```

Comment:
Why this change?

Reply:
This prevents you from deleting the active question in a game. This caused issues with the index and also the question does not auto remove.